src/main/java/org/opentripplanner/routing/algorithm/strategies/street_heuristic/Boundaries.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner.routing.algorithm.strategies.street_heuristic;
+
+import lombok.Getter;
+import org.opentripplanner.graph_builder.module.street_heuristic.SerializedStreetHeuristicData;
+
+import java.io.Serializable;
+
+class Boundaries implements Serializable {
+
+    private final double latMin, latMax, lonMin, lonMax;
+
+    @Getter
+    private final int width, height;
+
+    private final double cellHeight, cellWidth, latBegin, lonBegin, latEnd, lonEnd;
+
+    Boundaries(double latMin, double latMax, double lonMin, double lonMax, int width, int height) {
+        this.latMin = latMin;
+        this.latMax = latMax;
+        this.lonMin = lonMin;
+        this.lonMax = lonMax;
+        this.width = width;
+        this.height = height;
+        cellHeight = (latMax - latMin) / height;
+        cellWidth = (lonMax - lonMin) / width;
+        latBegin = latMin - cellHeight / 2;
+        lonBegin = lonMin - cellWidth / 2;
+        latEnd = latMax + cellHeight / 2;
+        lonEnd = lonMax + cellWidth / 2;
```

Comment:
```suggestion
        latEnd = latMax - cellHeight / 2;
        lonEnd = lonMax - cellWidth / 2;
```
Check if max is really pointing at point + 0.5 of width/height of cell after border

Reply:
it was my mistake - lonMin and lonMax are already proper borders